### PR TITLE
Hotfixes for scanning errors

### PR DESCRIPTION
**Pull request by nahojjjen**
Tweaks the scanning parameters for candy and cp to handle errors users have encountered.

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/OcrHelper.java [hunk 1]
```diff
@@ -509,12 +509,12 @@ private String getPokemonIdentifierFromImg(Bitmap pokemonImage) {
...
         }
 
-        Bitmap candyAmount = getImageCrop(pokemonImage, 0.65, 0.7, 0.15, 0.035);
+        Bitmap candyAmount = getImageCrop(pokemonImage, 0.60, 0.7, 0.20, 0.035);
         String hash = "candyAmount" + hashBitmap(candyAmount);
         String pokemonCandyStr = ocrCache.get(hash);
 
         if (pokemonCandyStr == null) {
-            candyAmount = replaceColors(candyAmount, true, 55, 66, 61, Color.WHITE, 200, true);
+            candyAmount = replaceColors(candyAmount, true, 55, 66, 61, Color.WHITE, 100, true);
```
Comment (NightMadness) [inline app/src/main/java/com/kamron/pogoiv/OcrHelper.java]: this should remove the candy completely
replaceColors(candyAmount, true, 68, 105, 108, Color.WHITE, 90, true);
color replace

File: app/src/main/java/com/kamron/pogoiv/OcrHelper.java [hunk 2]
```diff
@@ -509,12 +509,12 @@ private String getPokemonIdentifierFromImg(Bitmap pokemonImage) {
             return Optional.absent();
         }
 
-        Bitmap candyAmount = getImageCrop(pokemonImage, 0.65, 0.7, 0.15, 0.035);
+        Bitmap candyAmount = getImageCrop(pokemonImage, 0.60, 0.7, 0.20, 0.035);
```
Comment (NightMadness) [inline app/src/main/java/com/kamron/pogoiv/OcrHelper.java]: i'm proposing we go up a bit and higher and more height
getImageCrop(pokemonImage, 0.60, 0.695, 0.20, 0.038)
modified image1

PR value
changed

**Comment by NightMadness** [comment]
I'm proposing we go up a bit and higher and more height
getImageCrop(pokemonImage, 0.60, 0.695, 0.20, 0.038)
modified image1

PR value
changed

this should remove the candy completely
replaceColors(candyAmount, true, 68, 105, 108, Color.WHITE, 90, true);
color replace

**Comment by nahojjjen** [comment]
I havent tested your numbers but your images look good.

Make a PR with the change to this branch, and then I'll merge this branch to master.

**Comment by NightMadness** [comment]
Sure no problem

**Pull request merged**